ts/components/services/ServicesTab.tsx

```diff
@@ -0,0 +1,271 @@
+  sections: ReadonlyArray<ServicesSectionState>;
+  isRefreshing: boolean;
+  onRefresh: (hideToast?: boolean) => void; // tslint:disable-line bool-param-default
+  onServiceSelect: (service: ServicePublic) => void;
+  handleOnLongPressItem: () => void;
+  isLongPressEnabled: boolean;
+  updateOrganizationsOfInterestMetadata?: (
+    selectedItemIds: Option<Set<string>>
+  ) => void;
+  onItemSwitchValueChanged: (service: ServicePublic, value: boolean) => void;
+  tabOffset: Animated.Value;
+}>;
+
+type Props = OwnProps &
+  ReturnType<typeof mapStateToProps> &
+  ReturnType<typeof mapDispatchToProps> &
+  LightModalContextInterface;
+
+const styles = StyleSheet.create({
+  activeTextStyle: {
+    ...makeFontStyleObject(Platform.select, "600"),
+    fontSize: Platform.OS === "android" ? 16 : undefined,
+    fontWeight: Platform.OS === "android" ? "normal" : "bold"
+  },
+  textStyle: {
+    fontSize: customVariables.fontSizeSmall
+  },
+  organizationLogo: {
+    marginBottom: 0
+  },
+  icon: {
+    paddingHorizontal: (24 - 17) / 2 // (io-right icon width) - (io-trash icon width)
+  }
+});
+
+function renderOrganizationLogo(organizationFiscalCode: string) {
+  return (
+    <OrganizationLogo
+      logoUri={getLogoForOrganization(organizationFiscalCode)}
+      imageStyle={styles.organizationLogo}
+    />
+  );
+}
+
+function organizationContainsText(item: Organization, searchText: string) {
+  return isTextIncludedCaseInsensitive(item.name, searchText);
+}
+
+class ServicesTab extends React.PureComponent<Props> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {};
+    this.onSaveAreasOfInterest = this.onSaveAreasOfInterest.bind(this);
+    this.showChooserAreasOfInterestModal = this.showChooserAreasOfInterestModal.bind(
+      this
+    );
+    this.renderLocalQuickSectionDeletion = this.renderLocalQuickSectionDeletion.bind(
+      this
+    );
+  }
+  /**
+   * For tab Locals
+   */
+  private showChooserAreasOfInterestModal() {
+    const {
+      selectableOrganizations,
+      hideModal,
+      selectedOrganizations
+    } = this.props;
+
+    const OrganizationsList = createFactory(
+      ChooserListContainer<Organization>()
+    );
+    this.props.showModal(
+      <OrganizationsList
+        items={selectableOrganizations}
+        initialSelectedItemIds={some(new Set(selectedOrganizations || []))}
+        keyExtractor={(item: Organization) => item.fiscalCode}
+        itemTitleExtractor={(item: Organization) => item.name}
+        itemIconComponent={left(renderOrganizationLogo)}
+        onCancel={hideModal}
+        onSave={this.onSaveAreasOfInterest}
+        isRefreshEnabled={false}
+        matchingTextPredicate={organizationContainsText}
+        noSearchResultsSourceIcon={require("../../../img/services/icon-no-places.png")}
+        noSearchResultsSubtitle={I18n.t("services.areasOfInterest.searchEmpty")}
+      />
+    );
+  }
+
+  private onSaveAreasOfInterest(selectedFiscalCodes: Option<Set<string>>) {
```

Comment:
```suggestion
  private onSaveAreasOfInterest = (selectedFiscalCodes: Option<Set<string>>)  => {
```
As discussed we could avoid bind in the constructor and use arrow function to declare method